picard/ui/tablebaseddialog.py

```diff
@@ -113,27 +132,15 @@ def __init__(self, parent):
         self.create_table()
         self.finished.connect(self.save_state)
 
-    @abstractmethod
-    def get_value_for_row_id(self, row, value):
-        pass
-
-    def set_table_item(self, row, colname, obj, key, sortkey=None):
-        value = obj.get(key, "")
-        self.set_table_item_val(row, colname, value, sortkey)
-
-    def set_table_item_val(self, row, colname, value, sortkey=None):
-        # TODO: use Column.sortkey & align
-        # QVariant remembers the original type of the data
-        # matching comparison operator will be used when sorting
-        # get() will return a string, force conversion if asked to
-        if sortkey is None:
-            sortkey = sort_key(value, numeric=True)
-        item = SortableTableWidgetItem(sortkey)
-        item.setData(QtCore.Qt.ItemDataRole.DisplayRole, value)
```

Comment:
@phw not sure if there was a special reason to use `setData(DisplayRole)` here over `setText()` but I did the change in this PR.

Reply:
Using `setText` should be fine and should make no difference. `text()` and `setText()` are also just accessing the data with `DisplayRole`.